src/data/requirements/camouflages/blur.js

```diff
@@ -124,4 +124,22 @@ export default {
       type: 'kills_shortly_after_field_upgrade'
     }
   }
```

Comment:
Missing comma here too 😄 
```suggestion
  },
```